.ci/Jenkinsfile

```
@@ -0,0 +1,94 @@
+#!groovy
+/**
+ * Jenkins pipeline to build Corda DJVM
+ */
+
+/**
+ * Kill already started job.
+ * Assume new commit takes precendence and results from previous
+ * unfinished builds are not required.
+ * This feature doesn't play well with disableConcurrentBuilds() option
+ */
+@Library('existing-build-control')
+import static com.r3.build.BuildControl.killAllExistingBuildsForJob
+killAllExistingBuildsForJob(env.JOB_NAME, env.BUILD_NUMBER.toInteger())
+
+/**
+ * Sense environment
+ */
+boolean isMaster = (env.BRANCH_NAME =~ /^master$/)
+
+pipeline {
+    agent {
+        label 'k8s'
```

I don't believe this build needs to be started on an agent configured for K8S cluster.
```suggestion
        label 'standard'
```